packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,33 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
```

Comment:
Can you add new lines between the headers and content?

Reply:
Yes, you can add more lines. What are you thinking about?

Reply:
Oh I meant a new line like this:
```sh
### Installation

To proceed...
```

Reply:
Yeah right. Sorry I thought I wanted to add something